execution/genesis.json

```diff
@@ -16,88 +14,68 @@
 		"muirGlacierBlock": 0,
 		"berlinBlock": 0,
 		"londonBlock": 0,
-		"terminalBlockHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
 		"arrowGlacierBlock": 0,
 		"grayGlacierBlock": 0,
+		"mergeNetsplitBlock": 0,
+		"shanghaiTime": 1681393665,
```

Comment:
Confused by this, won't this stop working for any new run ? This is very specific to the genesis time